style.css

```diff
@@ -1,5 +1,5 @@
 body{
-  color: black;
```

Comment:
You need to write #000 it will help u in feature
  color: black;

Reply:
- [ ] Please, this project aims to learn how to do a code review and learn HTML and CSS simultaneously. Kindly make your suggestions /feedback clear and meaningful, so students understand what you said and why you need a change. Explain it with some references may be.

For example " It would be better to stick with the design, the nav bar background color in the temple `black` . Kindly fix :wrench:  the color in order to resemble the design. :+1: "